src/gl/opengl/vcShader.cpp

```diff
@@ -235,7 +245,7 @@ bool vcShader_GetSamplerIndex(vcShaderSampler **ppSampler, vcShader *pShader, co
   GLuint uID = glGetUniformLocation(pShader->programID, pSamplerName);
 
   if (uID == GL_INVALID_INDEX)
-    return false;
+    return true;
```

Comment:
This seems wrong?

Reply:
This is temporary - once we start using Shader Conductor for all of our shaders, this function won't be needed at all and can be changed to the DirectX implementation (it just returns `true`). We have something like `u_texture` in GLSL, but in HLSL it's `sampler0` and `texture0` and when this is converted using Shader Conductor it becomes `SPIRV_Cross_Combinedtexture0sampler0` -  we don't get this sampler index until we attempt to bind the texture, so we can't magic up this name in this function.